gremlin-go/driver/connection_test.go

```diff
@@ -133,20 +134,51 @@ func readCount(t *testing.T, g *GraphTraversalSource, label string, expected int
 func readUsingAnonymousTraversal(t *testing.T, g *GraphTraversalSource) {
 	results, err := g.V().Fold().
 		Project(testLabel, personLabel).
-			By(T__.Unfold().HasLabel(testLabel).Count()).
-			By(T__.Unfold().HasLabel(personLabel).Count()).
+		By(T__.Unfold().HasLabel(testLabel).Count()).
+		By(T__.Unfold().HasLabel(personLabel).Count()).
 		ToList()
 	assert.Nil(t, err)
-	assert.Equal(t,1, len(results))
+	assert.Equal(t, 1, len(results))
 	resultMap := results[0].GetInterface().(map[interface{}]interface{})
 	assert.Equal(t, int64(0), resultMap[testLabel])
 	assert.Equal(t, int64(len(getTestNames())), resultMap[personLabel])
 }
 
+func getEnvOrDefaultString(key string, defaultValue string) string {
+	// Missing value is returned as "".
+	value := os.Getenv(key)
+	if len(value) != 0 {
+		return value
+	}
+	return defaultValue
+}
+
+func getEnvOrDefaultBool(key string, defaultValue bool) bool {
+	value := getEnvOrDefaultString(key, "")
+	if len(value) != 0 {
+		boolValue, err := strconv.ParseBool(value)
+		if err != nil {
```

Comment:
Why returned boolValue when error has occurred?

Reply:
Oops, this was supposed to read err == nil. Error is repeated below, i will fix this..